projects/public-search/src/app/patron-profile/patron-profile-requests/patron-profile-request/patron-profile-request.component.html

```diff
@@ -16,69 +16,66 @@
-              }
-            }
-            <!-- PENDING OR ITEM IN TRANSIT FOR PICKUP -->
-            @if (record.metadata.state === 'PENDING' || record.metadata.state === 'ITEM_IN_TRANSIT_FOR_PICKUP') {
-              <i class="fa fa-ellipsis-h pr-1"  title="{{ 'pending' | translate }}" aria-hidden="true"></i>
-              <!-- TODO: Add expired date and remove "waiting" text
-              expected availability 00.00.0000  -->
-              {{ 'waiting' | translate }}
-            }
-            <!-- PENDING -->
-            @if (record.metadata.state === 'PENDING' && record.metadata.rank && record.metadata.rank > 0) {
-              <div [translate]="'(position {{ rank }} in waiting list)'" [translateParams]="{ rank: record.metadata.rank }"></div>
-            }
+        <public-search-patron-profile-document
+          [record]="record"
+          [showAdditionalInformation]="!isCollapsed"
+          [isAnimated]="true"
+        />
+        <dl class="metadata">
+          <dt translate>Pickup location</dt>
+          <dd>{{ record.metadata.pickup_name }}</dd>
+        </dl>
+        <dl class="metadata md:hidden">
+          <dt class="md:hidden" translate>Status</dt>
+          <dd>
+            <ng-container [ngTemplateOutlet]="status" />
           </dd>
         </dl>
-        <!-- ACTION BUTTONS ================================================= -->
-        <div class="col-lg-2 col-12 text-right">
-          @if (record.metadata.state === 'PENDING' || record.metadata.state === 'ITEM_IN_TRANSIT_FOR_PICKUP') {
-            <button
-              id="request-cancel-{{ record.metadata.pid }}"
-              type="button"
-              class="btn btn-sm btn-primary"
-              (click)="cancel()"
-              [disabled]="cancelInProgress"
-            >
-              @if (!cancelInProgress || actionDone) {
-                {{ 'Cancel' | translate }}
-              } @else {
-                <span class="spinner-border spinner-border-sm mr-1" role="status"></span>
-                {{ 'Cancel' | translate }}
-              }
-            </button>
-          } @else {
-            &nbsp;
-          }
-        </div>
       </div>
+    </div>
+        <!-- STATUS ========================================================= -->
+    <div class="md:col-2 col-12 hidden md:block">
+      <ng-container [ngTemplateOutlet]="status" />
+    </div>
+    <!-- ACTION BUTTONS ================================================= -->
+    <div class="md:col-2 col-12 flex justify-content-end">
+      @if (['PENDING', 'ITEM_IN_TRANSIT_FOR_PICKUP'].includes(record.metadata.state)) {
+        <p-button
+          id="request-cancel-{{ record.metadata.pid }}"
+          (onClick)="cancel()"
+          [disabled]="cancelInProgress"
+          [label]="'Cancel' | translate"
+          [icon]="!(!cancelInProgress || actionDone)? 'pi pi-spin pi-spinner': null"
```

Comment:
[nice-to-have enhancement] Add icon pi pi-cart-minus to this button and make it outline red?

Reply:
Sure? A cart on a cancel button? I would be better to use fontawsome, no?

Reply:
OK, we canuse pi-trash or fa-trash